server/src/components/ListItem.js

```diff
@@ -24,7 +24,9 @@ export default class ListItem extends Component {
   }
 
   copy () {
-    const copied = copy(JSON.stringify(this.props.item))
+    const { item } = this.props
+    const event = { event: item['x-github-event'], payload: item.body }
```

Comment:
Are you sure the event is 'x-github-event' and not 'X-GitHub-Event'?

See: https://developer.github.com/webhooks/#delivery-headers

Reply:
In this case I think it is lowercase because of how Node handles HTTP headers: https://nodejs.org/en/docs/guides/anatomy-of-an-http-transaction/#method-url-and-headers

> It's important to note here that all headers are represented in lower-case only, regardless of how the client actually sent them. This simplifies the task of parsing headers for whatever purpose.

Reply:
ah, should be good then 👍